docs/clinic-integration.md

```diff
@@ -0,0 +1,642 @@
+                        "lastUpdatedReason": [
+                            "SCHEMA_MIGRATION"
+                        ],
+                        "lastUploadDate": "2023-10-23T14:28:08.107Z"
+                    },
+                    "offsetPeriods": {
+                        "14d": {
+                            // omitted for brevity
+                        },
+                        "1d": {
+                            // omitted for brevity
+                        },
+                        "30d": {
+                            // omitted for brevity
+                        },
+                        "7d": {
+                            // omitted for brevity
+                        }
+                    },
+                    "periods": {
+                        "14d": {
+                            // omitted for brevity
+                        },
+                        "1d": {
+                            // omitted for brevity
+                        },
+                        "30d": {
+                            // omitted for brevity
+                        },
+                        "7d": {
+                            // omitted for brevity
+                        }
+                    },
+                    "totalHours": 1410
+                }
+            },
+            "tags": null,
+            "targetDevices": [
+                "onetouchverioiq"
+            ],
+            "updatedTime": "2023-10-23T14:18:15.829Z"
+        },
+    ],
+    "meta": {
+        "count": 2
+    }
+}
+```
+
+### Search for a Patient
+
+The API for searching for patients is the same as for enumerating patients, with the addition of a `search` parameter. It allows you to search the patient list for a match with a fuzzy search term that matches names, emails, or medical record numbers (MRNs).
+
+```shell
+curl -X GET -H "X-Tidepool-Session-Token: {access_token}" "https://app.tidepool.org/v1/clinics/{clinicId}/patients?offset={start}&limit={limit}&search={search_term}"
+```
+
+The resulting patient list has the same format as the previous example, omitted here for brevity.
+
+### Fetch Patient Record
+
+If you know the user ID of a patient, you can fetch their patient record directly.
+
+```shell
+curl -X GET -H "X-Tidepool-Session-Token: {access_token}" "https://app.tidepool.org/v1/clinics/{clinicId}/patients/{patientUserId}"
+```
+
+The resulting patient record has the same format as one element of the data array in the previous examples, omitted here for brevity.
+
+### Fetch Patient Data
+
+You can fetch the individual diabetes data points for any given patient by invoking the data API. The start and end dates are ISO8601 formatted dates, and the data type list is a comma-separated list of data types you want to fetch. The full list of data types is `cbg,smbg,basal,bolus,wizard,food,cgmSettings,deviceEvent,dosingDecision,insulin,physicalActivity,pumpSettings,reportedState,upload,water`.
```

Comment:
Can we refer to https://tidepool.redocly.app/reference/data.v1/data/datatype.v1 instead, so that we don't risk this list becoming outdated?